### DISCUSSION TITLE
CSV to HL7 in Mirth - Issue with extraneous information HL7 output

### DISCUSSION THREAD

**Discussion by njpowell218** [nextgenhealthcare/connect#6146] [Q&A] [answered]
I have a channel creating HL7 messages from a csv file which is working successfully with one exception.  In each output file I am getting the first line of extraneous information shown in **bold** below that is not part of my HL7 message.  I need to understand how this is getting there and how I can eliminate it.  
**Input: csv**
123456,012345678,Testerson,Testy,H,2020-05-01,36,T2025,,32,12,48628

**Output HL7**
**delimited|row|column1|123456column2|012345678column3|Testersoncolumn4|Testycolumn5|Hcolumn6|2020-05-01column7|36column8|T2025column9|column10|32column11|12column12|48628**
MSH||||SCDDSN|SCDDSN|DDSN|20240326130920||DFT^P03|3|P|2.3
EVN|P03|20240326130920|PND
PID|1|123456|123456||Testerson^Testy^H||||||||||||||012345678|
PV1|1|O|
FT1|1|||20200501||CG|T2025|||32||||||12||||36|

**Summary Data Types**
![1 -SummaryDataTypes](https://github.com/nextgenhealthcare/connect/assets/162730058/35265369-31c7-4bd1-bd08-02a266103d45)
**Source**
![1a-Source](https://github.com/nextgenhealthcare/connect/assets/162730058/33f3bd65-0369-43a0-92f1-6862820e1a6c)
**Destination**
![1b-Destination](https://github.com/nextgenhealthcare/connect/assets/162730058/c569bfe3-b60e-452c-8b19-2031e7abfba6)
**Destination Transformer**
![2a-DestinationTransformer](https://github.com/nextgenhealthcare/connect/assets/162730058/709099c3-8475-4032-a0ce-b78a8df0872f)
![2b-DestinationTransformer](https://github.com/nextgenhealthcare/connect/assets/162730058/f68057be-260e-4e78-8d1a-a54d29d69aea)

**HL7 Output File**
[123456_6_20240926.txt](https://github.com/nextgenhealthcare/connect/files/14763490/123456_6_20240926.txt)

Thank you
Nancy

**Comment by pacmano1** [comment, marked as answer]
You assign` tmp=msg`

Instead put a stub MSH segment in the outbound message template and remove that line.

**Comment by njpowell218** [comment]
Perfect.  That did it!  Thank you so much for the Super Fast Response!!!

**Comment by pacmano1** [comment]
Next time post to discussions first.   Your question was neither a bug nor feature request.

**Comment by njpowell218** [comment]
I am new to GitHub - First time posting so I appreciate the guidance!!
